org.entangle_photo.Manager.json

```diff
@@ -0,0 +1,153 @@
+{
+    "app-id": "org.entangle_photo.Manager",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "branch": "stable",
```

Comment:
```suggestion
```

Reply:
Does this mean that flathub calls `flatpak-builder` with `--default-branch=stable` ? Just to know so I can update my command to test locally.

Reply:
The App Requirements page answered, yes. https://github.com/flathub/flathub/wiki/App-Requirements#branches-and-runtimes